actions/constantsActions.js

```diff
@@ -0,0 +1,43 @@
+import fetch from 'isomorphic-fetch';
+import { HOST_URL } from '../actions';
+const url = '/api/constants';
+const REQUEST = 'yasp/constants/REQUEST';
+const OK = 'yasp/constants/OK';
+const ERROR = 'yasp/constants/ERROR';
+
+export const constantsActions = {
+  REQUEST,
+  OK,
+  ERROR,
+};
+
+const getConstantsRequest = () => ({
+  type: REQUEST,
+});
+const getConstantsOk = (payload) => ({
+  type: OK,
+  payload,
+});
+const getConstantsError = (payload) => ({
+  type: ERROR,
+  payload,
+});
+
+export const getConstants = (host = HOST_URL) => (dispatch) => {
+  getConstantsRequest();
+  return fetch(`${host}${url}`, { credentials: 'include' })
```

Comment:
technically this isn't needed except for metadata and would save a tiny bit of traffic